python/ray/tune/tests/test_sync.py

```diff
@@ -167,18 +167,20 @@ def trainable(config):
         def counter(local, remote):
             mock()
 
+        sync_config = tune.SyncConfig(upload_dir="test", sync_to_cloud=counter)
         tune.syncer.CLOUD_SYNC_PERIOD = 1
+        os.environ["TUNE_GLOBAL_CHECKPOINT_S"] = "0"
```

Comment:
should this be 0.5 or does it not matter?

Reply:
it should be ok as tests pass for me but i’ll leave a comment.